src/pal/src/init/pal.cpp

```diff
@@ -1130,14 +1130,10 @@ static LPWSTR INIT_FormatCommandLine (int argc, const char * const *argv)
 
 /*++
 Function:
-  INIT_FindEXEPath
-
-Abstract:
-    Determine the full, canonical path of the current executable by searching
-    $PATH.
+  INIT_ConvertEXEPath
```

Comment:
Could you please add "Abstract:" with short description of what the function does? It was present for the original function.